From 1303993b5b9a4195ebe4930a8cd345c875e7f25c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Tue, 9 Jun 2015 11:58:52 +0200
Subject: [PATCH 05/10] cmake: Do not install libgtest

libgtest is an internal test dependency, and should not be installed to
the live system as a part of LLVM.

Bug: http://llvm.org/bugs/show_bug.cgi?id=18341
---
 utils/unittest/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/utils/unittest/CMakeLists.txt b/utils/unittest/CMakeLists.txt
index c9a2cdd..bb4f667 100644
--- a/utils/unittest/CMakeLists.txt
+++ b/utils/unittest/CMakeLists.txt
@@ -37,6 +37,9 @@ if (PTHREAD_LIBRARY_PATH)
   list(APPEND LIBS pthread)
 endif()
 
+# Delay building until the tests pull it in, and avoid installing it
+set(EXCLUDE_FROM_ALL ON)
+
 add_llvm_library(gtest
   googletest/src/gtest-all.cc
 
-- 
2.9.3

